cli/features/tutorial.feature

```diff
@@ -28,7 +28,7 @@ Feature: Following the tutorial
       | Name of the application to create | todo-app           |
       | Description                       | A todo application |
       | Initial version                   |                    |
-      | ExoCom version                    | 0.16.0             |
+      | ExoCom version                    | latest             |
```

Comment:
I don't expect us to update Exocom that much. If it becomes a problem, we can automate the search and replace. It's pretty straightforward in Vim..  :)